homeassistant/components/azure_data_explorer/translations/en.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
This file has to be deleted, will be created by the translation service after it has been merged.

Reply:
The en.json file is fine to keep and add to your initial PR, since testing will be hard otherwise. Other locales need to come in via Lokalize + updates to en.json as well.

Reply:
so I keep it